### PR TITLE
[TASK] Update all dependencies

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,20 +26,20 @@
     },
     "require": {
         "php": "~7.2.0 || ~7.3.0 || ~7.4.0",
-        "typo3/cms-core": "^9.5.4 || ^10.2",
-        "typo3/cms-extbase": "^9.5 || ^10.2",
-        "typo3/cms-fluid": "^9.5 || ^10.2",
-        "typo3/cms-frontend": "^9.5 || ^10.2"
+        "typo3/cms-core": "^9.5.4 || ^10.4",
```
Comment (spoonerWeb) [inline composer.json:29]: Would use "^9.5 || ^10.4" like in the other requirements. Or is there a special reason for using minimum 9.5.4?

File: composer.json [hunk 2]
```diff
@@ -26,20 +26,20 @@
...
-        "squizlabs/php_codesniffer": "^3.5.3",
-        "typo3/cms-fluid-styled-content": "^9.5"
+        "codeception/codeception": "^4.1.5",
+        "friendsofphp/php-cs-fixer": "^2.16.3",
+        "helhum/typo3-composer-setup": "^0.5.6",
+        "helmich/typo3-typoscript-lint": "^2.1.1",
+        "nimut/testing-framework": "^5.0.3",
+        "phpunit/phpunit": "^7.5.20",
+        "squizlabs/php_codesniffer": "^3.5.5",
+        "typo3/cms-fluid-styled-content": "^9.5.18 || 10.4.3"
```
Comment (spoonerWeb) [inline composer.json:42]: Here I'd use "^9.5 || ^10.4" as well.

Comment (oliverklee) [reply composer.json:42]: Will do.

File: composer.json (hunk 1 above)
Comment (oliverklee) [reply composer.json:29]: I don't remember. I'll test this.

Comment (oliverklee) [reply composer.json]: Done.

File: composer.json (hunk 2 above)
Comment (oliverklee) [reply composer.json]: Done.

**Comment by oliverklee** [comment]
Updated, rebased and repushed.

**Pull request merged**